### ISSUE TITLE
add support for normal lita syntax

### ISSUE THREAD

**Issue by DimaVartanian** [jjasghar/lita-motivate#1]
hey! would be nice if the bot supported the usual syntax plugins use such as

"[bot_name] motivate someone"

as "!m someone" is a little more unusual from what I've seen of these scripts.

**Comment by jjasghar** [comment]
Yep, that's true. I basically wanted to port: http://motivate.im/ to lita. I can add it as a feature request at some point.
